src/slider.jsx

```diff
@@ -307,6 +307,18 @@ const Slider = React.createClass({
   },
 
 
+  // Needed to prevent text selection when dragging the slider handler.
+  // In the future, we should consider use <input type="range"> to avoid
+  // similar issues.
+  _toggleSelection(value) {
+    let body = document.getElementsByTagName('body')[0];
+    body.style['user-select'] = value;
```

Comment:
True! I forgot about looking better the style modules. I will fix it. I will also try to submit a PR sooner I can for the `<input type="range">`